src/team_lib/mhba/xccl_mhba_mkeys.c

```diff
@@ -157,29 +182,22 @@ static xccl_status_t poll_umr_cq(xccl_mhba_node_t *node)
 }
 
 // Execute the UMR WQE for populating the UMR's MasterMKey
-static xccl_status_t populate_mkey(xccl_mhba_team_t *team, int mem_access_flags,
-                                   struct mlx5dv_mkey *mkey, void *mkey_entries,
-                                   int repeat_count, int strided)
+static xccl_status_t
+populate_non_strided_mkey(xccl_mhba_team_t *team, int mem_access_flags,
```

Comment:
For non_strided do we still have the INLINE UMR limit?

Reply:
Yes - where is this relevant in the code?